<?xml version="1.0"?>

<launch>

  <!-- Note that axis IDs are those from the joystick message plus one, to be able to invert axes by specifiying either positive or negative axis numbers.-->
  <!-- Axis 2 from joy message thus has to be set as '3' or '-3'(inverted mode) below-->
  <node name="quadrotor_teleop" pkg="crazyflie_demo" type="quadrotor_teleop" output="screen">
    <param name="thrust_axis" value="-3"/>
    <param name="roll_axis" value="-1"/>
    <param name="pitch_axis" value="-2"/>
    <param name="mode_axis" value="-5"/>
    <param name="kp" value="6550"/>
    <param name="Pgain" value="70"/>
    <param name="Igain" value="40"/>
    <param name="Dgain" value="50"/>
    <param name="traverse_range" value="30"/>
    <remap from="kratos/pose" to="/vrpn_client_node/zm_kratos/pose" />

  </node>
</launch>
